storage/tests/system.py

```diff
@@ -952,3 +952,17 @@ def test_notification_w_user_project(self):
             self.assertEqual(notifications[0].topic_name, self.TOPIC_NAME)
         finally:
             notification.delete()
+
+
+class TestAnonymousClient(unittest.TestCase):
+
+    PUBLIC_BUCKET = 'gcp-public-data-landsat'
+
+    def test_access_to_public_bucket(self):
+        anonymous = storage.Client.create_anonymous_client()
+        bucket = anonymous.bucket(self.PUBLIC_BUCKET)
+        for blob in bucket.list_blobs():
+            print("Downloading blob: {}".format(blob.name))
```

Comment:
We shouldn't have print statements in a test (system or otherwise)

Reply:
Oops. ea8244b